style.css

```diff
@@ -0,0 +1,88 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  overflow-x: hidden;
+}
+
+.nav-bar {
+  display: flex;
+  background-color: #3c3a39;
+  height: 92px;
+  align-items: center;
+  justify-content: space-around;
+}
+
+.menu-list {
+  display: none;
+}
+
+#list-menu {
+  width: 80%;
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  width: 18px;
+  height: 2px;
+  background-color: #fff;
+  margin: 4px 0;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  color: #fff;
+  text-align: center;
+}
+
+.container {
+  margin: 0;
+  padding: 0;
+  min-width: 100%;
+}
+
+.intro {
+  display: flex;
+  flex-direction: column;
+  background-size: 130px auto;
+  padding: 130px 24px 100px;
+  background-color: #1c1a19;
+  justify-content: center;
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  background-image: url(/images/illustration.png);
+}
+
+.intro h1 {
+  margin: 0;
+  font-size: 56px;
+  font-family: 'Crete Round', serif;
```

Comment:
Please note that this font is not loading since you've not imported the font into your style sheet. You can do this with the following code snippet in your style.css as the first line of code.
```CSS
@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Lato:wght@100&family=Open+Sans:wght@300&family=Poppins&family=Roboto:ital,wght@0,700;1,100&display=swap');
```
## Figma
<img src="https://user-images.githubusercontent.com/16392046/192094400-24857f04-3582-4267-ba6a-7baa25eb3450.png" width=200 />

## Your Implementation

<img src="https://user-images.githubusercontent.com/16392046/192094478-c0954fdf-d399-439d-8180-713217a254c1.png" width=200 />